src/herbpy/herbrobot.py

```diff
@@ -56,7 +86,54 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
         accel_limits[self.left_arm.GetArmIndices()] = [ 2. ] * self.left_arm.GetArmDOF()
         accel_limits[self.right_arm.GetArmIndices()] = [ 2. ] * self.right_arm.GetArmDOF()
         self.SetDOFAccelerationLimits(accel_limits)
-        
+
+
+        # Determine always-on controllers
+
+        # hand controllers
+        if not left_hand_sim:
+            self.controllers_always_on.append('left_hand_controller')
+
+        if not right_hand_sim:
+            self.controllers_always_on.append('right_hand_controller')
+
+        # force/torque controllers
+        if not left_ft_sim or not right_ft_sim:
+            self.controllers_always_on.append('force_torque_sensor_controller')
+
+        if not left_ft_sim:
+            self.controllers_always_on.append('left_tare_controller')
+
+        if not right_ft_sim:
+            self.controllers_always_on.append('right_tare_controller')
+
+        # Set default manipulator controllers
+        # NOTE: head is ignored until TODO new Schunk head integrated
+        if not left_arm_sim:
+            controllers_manip.append(
+                'left_gravity_compensation_controller')
+        else:
+            self.left_arm.sim_controller = self.AttachController(name=self.left_arm.GetName(),
+                                                                 args='IdealController',
+                                                                 dof_indices=self.left_arm.GetArmIndices(),
+                                                                 affine_dofs=0,
+                                                                 simulated=True)
```

Comment:
Same as other place, I'll leave this as-is since I would just duplicated the exact code here.